packages/dhis2/src/tracker.js

```diff
@@ -0,0 +1,126 @@
+import { expandReferences } from '@openfn/language-common/util';
+import * as util from './Utils';
+
+/**
+ * State object
+ * @typedef {Object} Dhis2State
+ * @private
+ * @property data - The response body (as JSON)
+ * @property response - The HTTP response from the Dhis2 server (excluding the body)
+ * @property references - An array of all previous data objects used in the Job
+ */
+
+/**
+ * Options object
+ * @typedef {Object} RequestOptions
+ * @property {object} query - An object of query parameters to be encoded into the URL
+ * @property {object} headers - An object of all request headers
+ * @property {string} [apiVersion=42] - The apiVersion of the request. Defaults to 42.
+ */
+
+/**
+ * Import data into DHIS2 using the tracker endpoint.
+ * @public
+ * @function
+ * @param {string} strategy - The effect the import should have. Can either be CREATE, UPDATE, CREATE_AND_UPDATE and DELETE.
+ * @param {object} payload - The data to be imported.
+ * @param {RequestOptions} [options] - An optional object containing query,and headers for the request
+ * @state {Dhis2State}
+ * @returns {Operation}
+ * @example <caption>Import a trackedEntity resource</caption>
+ * tracker.import('CREATE', {
+ * trackedEntities: [
+ *   {
+ *     orgUnit: 'TSyzvBiovKh',
+ *     trackedEntityType: 'nEenWmSyUEp',
+ *     attributes: [
+ *       {
+ *         attribute: 'w75KJ2mc4zz',
+ *         value: 'Gigiwe',
+ *       },
+ *     ],
+ *   },
+ *  ],
+ * });
+ */
+function _import(strategy, payload, options = {}, callback = s => s) {
```

Comment:
We should add `@alias` to avoid docs looking like this 
<img width="446" alt="Screenshot 2025-04-24 at 2 35 41 PM" src="https://github.com/user-attachments/assets/15b9dc3f-bc13-4cdb-b197-5c8a26cfb8b9" />